nexus/src/external_api/http_entrypoints.rs

```diff
@@ -1188,7 +1188,7 @@ type IpPoolRangePaginationParams = PaginationParams<EmptyScanParams, IpNetwork>;
 
 /// List ranges for an IP pool
 ///
-/// Ranges are ordered by their first address.
+/// List ranges for an IP pool. Ranges are ordered by their first address.
```

Comment:
Is this first part needed? It's already captured above.

Reply:
The summary and description are not combined in the docs. At least not currently.